alchemy/src/resend/audience.ts

```diff
@@ -0,0 +1,100 @@
+import type { Context } from "../context.ts";
+import { Resource } from "../resource.ts";
+import { createResend, type ResendProps } from "./api.ts";
+
+export interface AudienceProps extends ResendProps {
+  /**
+   * The name of the audience.
+   * @default ${app}-${stage}-${id}
+   */
+  name?: string;
+}
```

Comment:
Audiences can't be adopted, but we do have a way to reference an existing one by ID.